src/csvcubed/cli/error_mapping.py

```diff
@@ -64,8 +75,12 @@ def friendly_error_mapping(error: ValidationError) -> str:
         CsvColumnUriTemplateMissingError: "The '{error.csv_column_name}' column definition is missing a 'cell_uri_template'; a suitable "
         "value could not be inferred.",
         DuplicateColumnTitleError: "There are multiple CSV columns with the title: '{error.csv_column_title}'.",
+        DuplicateMeasureError: "In the pivoted shape, each observation value column must use a unique measure. Affected columns: {error.column_names_concatenated}",
         EmptyQbMultiMeasureDimensionError: "A Measure column has been defined but no measures have been defined within it",
         EmptyQbMultiUnitsError: "A Unit column has been defined but no units have been defined within it",
+        HybridShapeError: "Mutliple observation value columns have been at the same time as a standard shape measure column defined.",
+        LinkedObsColumnDoesntExistError: "The '{error.attribute_column_title}' column has `describes_observations` set to '{error.alleged_obs_val_column_title}'. The column does not appear to exist.",
+        LinkedToNonObsColumnError: "The '{error.attribute_column_title}' column has `describes_observations` set to '{error.alleged_obs_val_column_title}'. This column does not represent observed values.",
```

Comment:
Have you missed `PivotedShapeMeasureColumnsExistError` here?

Reply:
Added doc page for `PivotedShapeMeasureColumnsExistError` + appended the index + appended to this `error_mapping`